plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -445,26 +444,29 @@ public static function add_component_settings( $settings ) {
 	 *
 	 * @param array $statuses Order statuses.
 	 * @return array formatted statuses.
+	 *
+	 * @deprecated migrate to \Automattic\WooCommerce\Internal\Admin\Settings instead.
 	 */
 	public static function get_order_statuses( $statuses ) {
-		$formatted_statuses = array();
-		foreach ( $statuses as $key => $value ) {
-			$formatted_key                        = preg_replace( '/^wc-/', '', $key );
-			$formatted_statuses[ $formatted_key ] = $value;
-		}
-		return $formatted_statuses;
+		wc_deprecated_function( __CLASS__ . '::' . __FUNCTION__, '9.9.0', '\Automattic\WooCommerce\Internal\Admin\Settings::get_order_statuses' );
+
+		return Settings::get_order_statuses( $statuses );
 	}
 
 	/**
 	 * Get all order statuses present in analytics tables that aren't registered.
 	 *
 	 * @return array Unregistered order statuses.
+	 *
+	 * @deprecated migrate to \Automattic\WooCommerce\Internal\Admin\Settings instead.
 	 */
 	public static function get_unregistered_order_statuses() {
+		wc_deprecated_function( __CLASS__ . '::' . __FUNCTION__, '9.9.0', 'Direct `\Automattic\WooCommerce\Internal\Admin\Settings` use' );
+
 		$registered_statuses   = wc_get_order_statuses();
 		$all_synced_statuses   = OrdersDataStore::get_all_statuses();
```

Comment:
I'm getting the following error when trying to call this function:

```
Fatal error: Uncaught Error: Class 'Automattic\WooCommerce\Internal\Admin\OrdersDataStore' not found
in /Users/joshua/Projects/woocommerce/plugins/woocommerce/src/Internal/Admin/Loader.php on line 467
```

We'll need to add `use Automattic\WooCommerce\Admin\API\Reports\Orders\DataStore as OrdersDataStore;` to this file.

Reply:
Addressed in 90d9a76f09c5fa845dfa421faa7cbc9f582261c5